kanvas/src/commonMain/kotlin/Transform.kt

```diff
@@ -51,6 +51,13 @@ public fun Transform.Scale.split(): Transform.InOrder = Transform.InOrder(
     Transform.Translate(-pivotX, -pivotY),
 )
 
+/** Converts a pivoted rotate into an ordered transformation of translate, rotate, translate. */
+public fun Transform.Rotate.split(): Transform.InOrder = Transform.InOrder(
```

Comment:
The stuff added in `commonMain` is used in the implementation. It could have been `internal` but I like to keep general utility stuff for implementing `Kanvas` public in case future implementations need them that aren't implemented directly in this module.